src/components/atoms/TextInput.vue

```diff
@@ -15,21 +17,33 @@
       @send-button="sendBtnText"
     />
     <button
+      ref="sendMsgBtn"
       class="float-right cursor-pointer"
       aria-label="Send message"
+      tabindex="0"
       @click="sendText"
+      @focus="onSendBtnFocus"
+      @mouseover="onSendBtnFocus"
```

Comment:
When I hover over with my mouse I don't get the focus button

Reply:
Did you write anything into the textbox? It won't display the focus icon unless there is text/the button is active

Reply:
Yes, I can show you what it looks like from my perspective when you have a minute

Reply:
Hahaha nvm... I was in mobile mode so no mouse...